semver/range_max.ts

```diff
@@ -44,15 +44,21 @@ function comparatorMax(comparator: Comparator): SemVer {
 }
 
 /**
+ * The maximum valid SemVer for a given range or INVALID
+ *
+ * @example Usage
+ * ```ts
+ * // This API is deprecated
```

Comment:
Perhaps, we can have two snippets in this example; one for the deprecated usage and one for the new API's usage, like a migration guide.